tutorials/napps/development_environment_setup.rst

```diff
@@ -38,16 +38,67 @@ Installing required dependencies
 ********************************
 
 In order to start using and coding with Kytos, you need a few required
-dependencies. One of them is Python 3.6 or greater.
+dependencies. One of them is Python 3.9.x
 
+
```

Comment:
@usmanazFIU could you please update on which Ubuntu version you tested? It was 20.04 LTS you mentioned right? 

This part here `We tested this tutorial on Ubuntu...`

Reply:
ok sure

Reply:
yes 20.04 LTS